tutor/commands/jobs.py

```diff
@@ -124,14 +124,14 @@ def create_user_template(
 @click.option(
     "-r",
     "--repo",
-    default="https://github.com/openedx/edx-demo-course",
+    default="https://github.com/openedx/openedx-demo-course",
     show_default=True,
     help="Git repository that contains the course to be imported",
 )
 @click.option(
     "-d",
     "--repo-dir",
-    default="",
+    default="demo-course/course",
```

Comment:
This change will cause some existing commands to fail, when users try to import a different custom course that is not stored in a subdirectory:

    tutor local do importdemocourse --repo https://github.com/me/mycourse

I wonder if we could simplify the CLI by auto-detecting the location of the "course.xml" file? This would make the import script much more complex. But I see no other option if we want to preserve ease-of-use and compatibility with the new demo course structure.

Reply:
Or maybe, for now, if the repo path ends with openedx-demo-course, treat the default as demo-course/course, otherwise ""?

Reply:
@regisb I am OK with that complexity. I'll implement this logic:

* If the `--repo-dir` argument is provided, use it.
* Else:
    * `find` the course.xml file
    * If there is exactly one course.xml, then use that directory.
    * Else if there are 0 or 2+ course.xml files, raise an exception.

@DawoudSheraz The old and new demo course are both stored in the same repo, "openedx-demo-course". "edx-demo-course" has redirected to "openedx-demo-course" a couple years.